### PR TITLE
K8s Improvements

### PR DESCRIPTION
**Pull request by jorge07**
Deps update
Logs to Stdout/Stderr
Helm3

### CODE REVIEW

**Comment by jorge07** [comment]
Nope. It shouldn't. Just forgot to add it to the gitignore. Good catch thanks!

File: makefile [hunk 1]
```diff
@@ -59,7 +59,7 @@ layer: ## Check issues with layers
 .PHONY: db
 db: ## recreate database
 		$(compose) exec -T php sh -lc './bin/console d:d:d --force || true'
-		$(compose) exec -T php sh -lc './bin/console d:d:c'
+		$(compose) exec -T php sh -lc './bin/console d:d:c || true'
```
Comment (cv65kr) [inline makefile:62]: Maybe add here `--if-not-exists` parameter. WDYT?

File: makefile [hunk 2]
```diff
@@ -59,7 +59,7 @@ layer: ## Check issues with layers
 .PHONY: db
 db: ## recreate database
 		$(compose) exec -T php sh -lc './bin/console d:d:d --force || true'
```
Comment (cv65kr) [inline makefile:61]: Maybe add here `--if-exists` parameter. WDYT?

File: makefile [hunk 3]
```diff
@@ -87,9 +87,13 @@ wait-for-elastic: ## Health check for elastic
...
 minikube:
 	@eval $$(minikube docker-env); \
-	docker-compose -f docker-compose.prod.yml build; \
-	helm dep up etc/artifact/chart
-	helm install --name cqrs etc/artifact/chart
+	docker-compose -f etc/artifact/docker-compose.yml build; \
+	helm dep up etc/artifact/chart; \
+	helm upgrade -i cqrs etc/artifact/chart
+
+htemplate:
```
Comment (cv65kr) [inline makefile:94]: `.PHONY` is not needed here?

**Comment by jorge07** [comment]
@cv65kr great feedback, thanks! All applied

**Pull request merged**